### PR TITLE
Teradata jdbc connector

### PR DESCRIPTION
**Pull request by ashishtadose**
Test plan - (Please fill in how you tested your changes)

Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.

Fill in the release notes towards the bottom of the PR description.
See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.

```
== RELEASE NOTES ==

General Changes
* ...
* ...

Hive Changes
* ...
* ...
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by dborkar** [comment]
@beinan something you can take a look at perhaps?

File: presto-teradata/repo/com/teradata/terajdbc4/16.20/terajdbc4-16.20.pom [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (beinan) [inline presto-teradata/repo/com/teradata/terajdbc4/16.20/terajdbc4-16.20.pom:1]: any reason not using the jar from the public maven repo?
https://mvnrepository.com/artifact/com.teradata.jdbc/terajdbc4/16.20.00.10

**Review by beinan — COMMENTED** [review]
Can I have more context why we need a separate connector for teradata jdbc connection?  I think the existing jdbc connector could also do the job.  Any further plan?

File: presto-teradata/repo/com/teradata/terajdbc4/16.20/terajdbc4-16.20.pom (hunk 1 above)
Comment (ashishtadose) [reply presto-teradata/repo/com/teradata/terajdbc4/16.20/terajdbc4-16.20.pom:1]: @beinan I had tried it out first, this artifact is not present central maven repository.
Even the ICM repo mentioned doesn't have it.

Comment (ashishtadose) [reply presto-teradata/repo/com/teradata/terajdbc4/16.20/terajdbc4-16.20.pom:1]: Is there any generic JDBC connector that can be leveraged? I couldn't find one

**Pull request closed without merging**